src/btree/bt_sync.c

```diff
@@ -227,6 +229,10 @@ __wt_sync_file(WT_SESSION_IMPL *session, WT_CACHE_OP syncop)
         __wt_gen_next_drain(session, WT_GEN_EVICT);
         btree->syncing = WT_BTREE_SYNC_RUNNING;
 
+        /* Add in lookaside reconciliation for standard files. */
+        if (!F_ISSET(btree, WT_BTREE_LOOKASIDE) && !WT_IS_METADATA(btree->dhandle))
+            rec_flags |= WT_REC_LOOKASIDE;
```

Comment:
This flag is actually saying "is this the lookaside or metadata" - could we either skip using a flag, or call the flag something more in line with that?

Reply:
I would prefer to keep the flag as is.

Reply:
I would like to update the comment with a TBD as " Remove the WT_BTREE_LOOKASIDE flag, and assert that no updates from lookaside are copied to lookaside recursively". A way to verify of design idea and to check the fallout if any.